lua/gk.lua

```diff
@@ -15,8 +15,50 @@ return function (net_conf, numa_table)
 	local ggu_lcore = table.remove(gk_lcores)
 	gatekeeper.gk_assign_lcores(gk_conf, gk_lcores)
 
+	gk_conf.max_num_ipv4_rules = 1024
+	gk_conf.num_ipv4_tbl8s = 256
+	gk_conf.max_num_ipv6_rules = 1024
+	gk_conf.num_ipv6_tbl8s = 65536
+
+	local fib_entries = {
+		{
+			["ip_addr"] = "187.73.32.0",
+			["prefix_len"] = 30,
+			["action"] = gatekeeper.c.GK_FWD_GRANTOR,
+			["grantor"] = "10.0.0.3",
+		},
+
+		{
+			["ip_addr"] = "187.73.33.0",
+			["prefix_len"] = 30,
+			["action"] = gatekeeper.c.GK_DROP,
+		},
+	}
+
+	local grantors = {
```

Comment:
Actually, I considered this issue before. I put the grantors separately, since this is easier for the administrators to maintain all the grantor servers in a single place, also, in the ``fib_entries`` list, multiple entries may share the same grantor server address.